modbus_rtu_block.py

```diff
@@ -8,17 +8,6 @@
 from nio.signal.base import Signal
 from nio.properties import StringProperty, IntProperty, \
     Property, VersionProperty, SelectProperty, PropertyHolder, ObjectProperty
-from nio.block.mixins.retry.strategy import BackoffStrategy
-
-
-class SleepBackoffStrategy(BackoffStrategy):
```

Comment:
This backoff strategy is not selectable in block config, and is over-ridden with a default strategy and never applied. furthermore, it is not the desired behavior.